pkg/get/tools.go

```diff
@@ -1823,5 +1823,23 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 		{{.Name}}-{{$osStr}}-latest`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:       "kumahq",
+			Repo:        "kuma",
+			Name:        "kumactl",
+			Version:     "1.3.1",
+			Description: "kumactl is a CLI to interact with Kuma and its data",
+			URLTemplate: `
```

Comment:
Looks good. We will need a unit test adding though. I don't mind if you do that as commit 2 or reset into one commit.

Reply:
I added some test for darwin x64 and linux x64 (only these arch + os combos are supported).